scripts/plotting/create_num_demos_plots.py

```diff
@@ -135,7 +134,7 @@ def _main() -> None:
                                 label=label,
                                 marker=marker)
                 # Automatically make x ticks integers for certain X KEYS.
-                if x_key in ("CYCLE", "NUM_TRANSITIONS"):
+                if x_key in ("CYCLE", "NUM_OFFLINE_TRANSITIONS"):
```

Comment:
will this script still work correctly? it looks like you didn't add `NUM_OFFLINE_TRANSITIONS` to `X_KEY_AND_LABEL` above

Reply:
fixed